src/transformers/commands/add_new_model.py

```diff
@@ -94,10 +94,12 @@ def run(self):
 
         lowercase_model_name = configuration["lowercase_modelname"]
         pytorch_or_tensorflow = configuration["generate_tensorflow_and_pytorch"]
+        generate_flax = configuration["generate_flax"]
```

Comment:
could we maybe remove this command and change the `"generate_tensorflow_and_pytorch"` command to `"generate_tensorflow_pytorch_and_flax"`?

Reply:
So ideally after `"Select tokenizer type:"` we would have:

```
"Select generate_tensorflow_pytorch_and_flax":
1 - PyTorch, TensorFlow and Flax
2 - PyTorch
3 - TensorFlow
4 - Flax
Choose from 1, 2, 3, 4 [1]:
```

and then remove the question:
```
Select generate_flax:
```
Otherwise it won't really be possible to generate flax without having to generate either PyTorch and/or Tensorflow

Reply:
Let me know if this doesn't make sense...otherwise this PR already looks great :-)

Reply:
Yes this is one way to do it. But this way we won't be able to generate only 2 of them. Of course we could add 3 more options for that.

```
"Select generate_tensorflow_pytorch_and_flax":
1 - PyTorch, TensorFlow and Flax
2 - PyTorch and TensorFlow
3 - PyTorch and Flax
4 - TensorFlow and Flax
5 - PyTorch
6 - TensorFlow
7 - Flax
```

Or we can make it 3 options, and check in `src/transformers/commands/add_new_model.py` if at least one is selected.
```
Select generate_pytorch:
Select generate_tensorflow:
Select generate_flax:
```
Which would you prefer? I have no strong opinion here.

Reply:
I fixed this accordingly. I will leave the final touches to you. Thank you for your review and help! :-)